### ISSUE TITLE
upgrade to polkadot v1.0.0

### ISSUE THREAD

**Issue by nakul1010** [interlay/interbtc#1148]
**Is your feature request related to a problem? Please describe.**
Upgrading to the latest version is a proactive approach to ensure that our network stays up-to-date with the latest developments and advancements. 

**Describe the solution you'd like**
All the para chain dependencies need to be upgraded to the latest [Polkadot v1.0.0](https://github.com/paritytech/polkadot/releases/tag/v1.0.0) release 

**Additional context**
Update [Xcm-simulator](https://github.com/shaunxw/xcm-simulator) to the latest release as it is one of the dependencies.

**Comment by nakul1010** [comment]
This issue is been blocked by xcm-simulator issue [here](https://github.com/shaunxw/xcm-simulator/issues/54)

**Comment by nakul1010** [comment]
The xcm-simulator is no more a blocker as it is already been merged into the cumulus repo [here](https://github.com/paritytech/cumulus/tree/master/xcm/xcm-emulator).
